src/test/scala/io/iohk/ethereum/jsonrpc/NetServiceSpec.scala

```diff
@@ -0,0 +1,50 @@
+package io.iohk.ethereum.jsonrpc
+
+import java.net.InetSocketAddress
+
+import akka.actor.ActorSystem
+import akka.agent.Agent
+import akka.testkit.TestProbe
+import io.iohk.ethereum.crypto
+import io.iohk.ethereum.jsonrpc.NetService.{ListeningRequest, ListeningResponse, PeerCountRequest, PeerCountResponse}
+import io.iohk.ethereum.network.{PeerActor, PeerManagerActor}
+import io.iohk.ethereum.network.PeerManagerActor.Peer
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.Status
+import io.iohk.ethereum.utils.{NodeStatus, ServerStatus}
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class NetServiceSpec extends FlatSpec with Matchers with MockFactory {
+
+  "NetService" should "return handshaked peer count" in new TestSetup {
```

Comment:
I realise `version` is trivial but I would still have it tested here, and in `JsonRpcControllerSpec`